crates/swc_css_lints/src/rules/font_family_no_duplicate_names.rs

```diff
@@ -0,0 +1,167 @@
+use std::ops::Deref;
+
+use ahash::AHashSet;
+use serde::{Deserialize, Serialize};
+use swc_css_ast::*;
+use swc_css_visit::{Visit, VisitWith};
+
+use crate::rule::{visitor_rule, LintRule, LintRuleContext};
+
+#[derive(Debug, Clone, Default, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct FontFamilyNoDuplicateNamesConfig {
+    ignore_font_family_names: Option<Vec<String>>,
+}
+
+pub fn font_family_no_duplicate_names(
+    ctx: LintRuleContext<FontFamilyNoDuplicateNamesConfig>,
+) -> Box<dyn LintRule> {
+    let ignored = ctx
+        .config()
+        .ignore_font_family_names
+        .clone()
+        .unwrap_or_default();
+    visitor_rule(ctx.reaction(), FontFamilyNoDuplicateNames { ctx, ignored })
+}
+
+#[derive(Debug, Default)]
+struct FontFamilyNoDuplicateNames {
+    ctx: LintRuleContext<FontFamilyNoDuplicateNamesConfig>,
+    ignored: Vec<String>,
+}
+
+impl Visit for FontFamilyNoDuplicateNames {
+    fn visit_declaration(&mut self, declaration: &Declaration) {
+        match &declaration.name {
+            DeclarationName::Ident(Ident { value, .. })
+                if value.eq_str_ignore_ascii_case("font-family") =>
+            {
+                declaration.value.iter().fold(
+                    AHashSet::with_capacity(declaration.value.len()),
+                    |mut fonts, item| {
+                        let font = match item {
+                            ComponentValue::Ident(Ident { value, .. }) => {
```

Comment:
There is potential problem, font can be sequence of ident, i.e. `font: Foo Bar, bar foo`, it is valid due legacy 

> Font family names other than generic families must either be given quoted as [strings,](https://www.w3.org/TR/CSS21/syndata.html#strings) or unquoted as a sequence of one or more [identifiers.](https://www.w3.org/TR/CSS21/syndata.html#value-def-identifier)

https://www.w3.org/TR/css-fonts-3/#family-name-value